tests/lldp/test_lldp.py

```diff
@@ -18,6 +20,73 @@ def lldp_setup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, patch_lldpct
     unpatch_lldpctl(localhost, duthost)
 
 
+@pytest.fixture(scope="function")
+def restart_orchagent(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    asic = duthost.asic_instance(enum_frontend_asic_index)
+    container_name = asic.get_docker_name("swss")
+    program_name = "orchagent"
+
+    logger.info("Restarting program '{}' in container '{}'".format(program_name, container_name))
+
+    duthost.shell("sudo config feature autorestart {} disabled".format(container_name))
+    _, program_pid = get_program_info(duthost, container_name, program_name)
+    kill_process_by_pid(duthost, container_name, program_name, program_pid)
+    is_running = is_container_running(duthost, container_name)
+    pytest_assert(is_running, "Container '{}' is not running. Exiting...".format(container_name))
+    duthost.shell("docker exec {} supervisorctl start {}".format(container_name, program_name))
+    yield
+    duthost.shell("sudo config feature autorestart {} enabled".format(container_name))
+
+
+def get_program_info(duthost, container_name, program_name):
+    """
```

Comment:
same function get_program_info is also in tests/common/helpers/dut_utils.py, for those basic functional functions, I recommend to move it to common folder. In test script, we just need to focus on test case scenarios. It can also make the script more readably.